### PR TITLE
float sqrt

### PR DESCRIPTION
**Pull request by Ethanout**

### CODE REVIEW

**Review by Ethanout — COMMENTED** [review]
float sqrt

File: datapacks/Bookshelf/data/bs.math/functions/sqrt/__main__.mcfunction [hunk 1]
```diff
@@ -1,34 +1,13 @@
...
-scoreboard players operation #math.sqrt.stemp_2 bs.data /= @s bs.out.0
-scoreboard players operation @s bs.out.0 += #math.sqrt.stemp_2 bs.data
-scoreboard players operation @s bs.out.0 /= 2 bs.const
-scoreboard players operation #math.sqrt.stemp_3 bs.data /= @s bs.out.0
-scoreboard players operation @s bs.out.0 += #math.sqrt.stemp_3 bs.data
-scoreboard players operation @s bs.out.0 /= 2 bs.const
+# is E >= 64
+execute store result score #check3 bs.data run data get storage bs.in math.sqrt 0.0000000000000000000542101086242752217003726400434970855712890625
+# break if negative
+execute if score #check bs.data matches ..-1 run return -1
```
Comment (Leirof) [inline datapacks/Bookshelf/data/bs.math/functions/sqrt/__main__.mcfunction:4]: According to the convention, `#check` should be renamed in `#math.sqrt.check`. Same for the other fake players

File: datapacks/Bookshelf/data/bs.math/functions/sqrt/__int__.mcfunction
Comment (Leirof) [inline datapacks/Bookshelf/data/bs.math/functions/sqrt/__int__.mcfunction:1]: This feature should be renamed `isqrt` and be place in a separated folder

File: datapacks/Bookshelf/data/bs.math/functions/sqrt/__main__.mcfunction
Comment (Leirof) [inline datapacks/Bookshelf/data/bs.math/functions/sqrt/__main__.mcfunction:1]: Missing header and the main file should have the same name as the feature, so here `sqrt`

**Review by Leirof — CHANGES_REQUESTED** [review]
Missing documentation

File: datapacks/Bookshelf/data/bs.math/functions/sqrt/__int__.mcfunction
Comment (aksiome) [reply datapacks/Bookshelf/data/bs.math/functions/sqrt/__int__.mcfunction:1]: I we rename it `isqrt` we also need to round the result to the lowest integer!

File: datapacks/Bookshelf/data/bs.math/functions/sqrt/__main__.mcfunction
Comment (Ethanout) [reply datapacks/Bookshelf/data/bs.math/functions/sqrt/__main__.mcfunction:1]: hmmm do u mean the __main__.mcfunction should be named as __sqrt__.mcfunction

Comment (Leirof) [reply datapacks/Bookshelf/data/bs.math/functions/sqrt/__main__.mcfunction:1]: Yep, it's a detail, but as @theogiraudet said, when we are working on several features, having a lot of "main" tabs in VSC is confusing ^^'

**Review by Leirof — CHANGES_REQUESTED** [review]
Missing documentation

**Comment by aksiome** [comment]
Please do not merge it into the master branch, i'll merge it myself into the rework of the math module ^^

**Comment by aksiome** [comment]
Merged into math module rework: #114 😉

**Pull request closed without merging**